src/libraries/System.Private.CoreLib/src/System/Runtime/InteropServices/MinimumOSAttribute.cs

```diff
@@ -0,0 +1,44 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Runtime.InteropServices
+{
+
+    /// <summary>
+    /// Records the minimum platform that is required in order to the marked thing.
+    /// <list type="bullet">
+    /// <item>
+    /// <description>When applied to an assembly, it means the entire assembly cannot be called
+    /// into on earlier versions. It records the TargetPlatformMinVersion property.</description>
+    /// </item>
+    /// <item>
+    /// <description> When applied to an API, it means the API cannot be called from an earlier
+    /// version.</description>
+    /// </item>
+    /// </list>
+    ///
+    /// In either case, the caller can either mark itself with MinimumPlatformAttribute
+    /// or guard the call with a platform check.
+    ///
+    /// The attribute can be applied multiple times for different operating systems.
+    /// That means the API is supported on multiple operating systems.
+    ///
+    /// A given platform should only be specified once.
+    /// </summary>
```

Comment:
```suggestion
    /// <summary>
    /// Records the operating system (and minimum version) that supports an API. Multiple attributes can be
    /// applied to indicate support on multiple operating systems.
    /// </summary>
    /// <remarks>
    /// Callers can apply a <see cref="System.Runtime.InteropServices.MinimumOSPlatformAttribute" />
    /// or use guards to prevent calls to APIs on unsupported operating systems.
    ///
    /// A given platform should only be specified once.
    /// </remarks>
```